rpcservers/neutrino/neutrino_server.go

```diff
@@ -242,37 +245,39 @@ func (s *Server) AddPeer(ctx context.Context,
 //
 // NOTE: Part of the NeutrinoKitServer interface.
 func (s *Server) DisconnectPeer(ctx context.Context,
-	in *DisconnectPeerRequest) (*DisconnectPeerResponse, error) {
+	in *neutrinorpc.DisconnectPeerRequest) (
+	*neutrinorpc.DisconnectPeerResponse, error) {
```

Comment:
nit: break before `error` instead.

Reply:
Won't comment on this further, I think you get the idea.

Reply:
All I can say about this is, gofmt stops so many disagreements between programmers, but where it fails to prescribe a form, peeople get into debates about it. I wish gofmt had rules about line length and tab sizes too. This is the first time I have worked on a project that prescribes the old school 8 character tab which really is a good thing if it forces you to break your functions into smaller units if there is that many layers of decisions, but it makes even this little task here a serious nit full experience haha.